README.md

```diff
@@ -130,7 +133,42 @@ To host your own Judge0 API you need to install [Docker](https://docs.docker.com
 
 6. Open `http://<IP OF YOUR SERVER>:3000` in your browser.
 
+## Carefully in Windows
+
+If you want to run Judge0 API on Windows, you have to change file `docker-compose.yml` .
```

Comment:
There is no `docker-compose.yml`. There are: `docker-compose.prod.yml`, `docker-compose.prod.https.yml` and `docker-compose.dev.yml`. If on Windows you need to change compose file you are using.